app/src/main/res/values-es/strings.xml

```diff
@@ -77,29 +58,99 @@
     <string name="setting_theme_dark">Oscuro moderno</string>
     <string name="setting_theme_classic_dark">Oscuro clásico</string>
     <string name="setting_theme_black">Negro moderno</string>
     <string name="setting_theme_classic_black">Negro clásico</string>
 
+    <string name="wind_direction_format_none">Sin indicadores</string>
+    <string name="wind_direction_format_arrows">Flechas</string>
+    <string name="wind_direction_format_abbreviations">Abreviaturas</string>
+    <string name="setting_displayDecimalZeroes">Mostrar ceros decimales</string>
+
+    <!-- Settings/Data -->
+    <string name="last_update">Última actualización:\n%s</string>
+    <string name="last_update_widget">\nÚltima actualización: %s</string>
+    <string name="settings_title_data">Datos</string>
+    <string name="setting_refreshInterval">Intervalo de actualización en segundo plano</string>
+    <string name="settings_none">Ninguno</string>
+    <string name="settings_15min">15 min</string>
+    <string name="settings_30min">30 min</string>
+    <string name="settings_1hour">1 h</string>
+    <string name="settings_2hour">2 h</string>
+    <string name="settings_6hour">6 h</string>
+    <string name="settings_12hour">12 h</string>
+    <string name="settings_24hour">24 h</string>
+    <string name="setting_apiKey">Clave de la API</string>
+    <string name="setting_apiKey_summary">Clave personalizada de la API para consultas en OpenWeatherMap</string>
+
+    <!-- Settings/Date format -->
+    <string name="error_dateFormat">ERROR EN FORMATO DE FECHA</string>
+    <string name="setting_dateFormat">Formato de fecha</string>
+    <string name="setting_dateFormatCustom">Formato de fecha personalizado</string>
+    <string name="setting_dateFormatCustom_summary">Elija la cadena SimpleDateFormat personalizada a utilizar</string>
+    <string name="setting_simple_string_formatting_intro">Puede usar los siguientes códigos para construir un formato de fecha y hora personalizado</string>
+    <string name="setting_available_simple_string_codes">yYMwWDdFEuaHkKhmsSzZX</string>
+    <string name="setting_example_simple_string">EEEE, d MMM yyyy h:mm:ss a, zzzz</string>
+    <string name="setting_example_date_format">Miércoles, 4 Jul 2001 12:08:32 PM, Tiempo del Pacífico con horario de verano</string>
     <string name="dialog_ok">Aceptar</string>
     <string name="dialog_cancel">Cancelar</string>
 
+    <!-- Errors -->
     <string name="msg_err_parsing_json">Error al analizar JSON.</string>
     <string name="msg_connection_problem">Hay un problema con su conexión a Internet.</string>
     <string name="msg_connection_not_available">Conexión no disponible.</string>
     <string name="msg_city_not_found">No se encuentra la ciudad especificada.</string>
-
+    <string name="msg_too_many_requests">Demasiadas solicitudes. Inténtelo más tarde.</string>
+
+    <!-- Weather icons -->
+    <string name="weather_cloudy" translatable="false">&#xf013;</string>
```

Comment:
do we need these lines? 125-151? as they are not translated, I was under the impression we should leave them out?

Reply:
Strings removed.